test/integration/evictions/evictions_test.go

```diff
@@ -216,14 +221,16 @@ func TestConcurrentEvictionRequests(t *testing.T) {
 		_, err := clientSet.Core().Pods(ns.Name).Get(podName, metav1.GetOptions{})
 		if !errors.IsNotFound(err) {
 			t.Errorf("Pod %q is expected to be evicted", podName)
+			if err == nil {
```

Comment:
it might be slightly cleaner (a little less cognitive overhead) to use a `switch` statement rather than testing the `if` statements